jetstream/cli.py

```diff
@@ -1045,3 +1096,250 @@ def ensure_enrollments(
+            type=experiment.type if experiment else "v6",
+            status="Live",
+            start_date=start_date - timedelta(days=3),  # subtract enrollment days
+            end_date=end_date,
+            proposed_enrollment=enrollment_period,
+            branches=experiment.branches if experiment else [Branch(slug="control", ratio=1)],
+            reference_branch=experiment.reference_branch if experiment else "control",
+            is_high_population=False,
+            app_name=platform,
+            app_id=PLATFORM_CONFIGS[platform].app_id,
+            outcomes=experiment.outcomes if experiment else [],
+            enrollment_end_date=None,
+        )
+
+        spec = AnalysisSpec.default_for_experiment(experiment, config_getter.configs)
+        if external_spec := config_getter.spec_for_experiment(experiment.normandy_slug):
+            spec.merge(external_spec)
+
+        config = spec.resolve(experiment=experiment, configs=config_getter.configs)
+
+        # generated sampled enrollment query
+        if generate_population:
+            spec.experiment.enrollment_query = sampled_enrollment_query(
+                start_date, config, population_sample_size
+            )
+
+            # update dates
+            spec.experiment.start_date = (start_date - timedelta(days=3)).strftime("%Y-%m-%d")
+            spec.experiment.end_date = end_date.strftime("%Y-%m-%d")
+            spec.experiment.enrollment_period = enrollment_period
+
+            # set experiments_column_type to none for all data sources used in the experiment;
+            # this needs to be done otherwise all clients will be filtered out since none
+            # are enrolled in the experiment
+            for _, summaries in config.metrics.items():
+                for summary in summaries:
+                    ds = summary.metric.data_source
+
+                    if ds.name in spec.data_sources.definitions:
+                        spec.data_sources.definitions[ds.name].experiments_column_type = "none"
+                    else:
+                        spec.data_sources.definitions[ds.name] = DataSourceDefinition(
+                            name=ds.name,
+                            from_expression=ds.from_expression,
+                            client_id_column=ds.client_id_column,
+                            submission_date_column=ds.submission_date_column,
+                            default_dataset=ds.default_dataset,
+                            build_id_column=ds.build_id_column,
+                            friendly_name=ds.friendly_name,
+                            description=ds.description,
+                            experiments_column_type="none",
+                        )
+
+        # log to a table in the temporary dataset, will be displayed on the Looker dashboard
+        log_config = LogConfiguration(
+            log_project_id=project_id,
+            log_dataset_id=dataset_id,
+            log_table_id=f"logs_{table}",
```

Comment:
It could be useful. I opened https://github.com/mozilla/jetstream/issues/1658 as a follow-up since adding custom prefixes for tables would require quite a few changes